### PR TITLE
Feature API

### PR DESCRIPTION
**Pull request by umarahgul**
In this PR:

1. Refresh button is functional 
2. Submit button lets user submit game name and score
3. RESTful Api leaderboard is used to fetch and response  
5.  async and await JavaScript features to consume the API

### CODE REVIEW

**Review by Ibizugbe — COMMENTED** [review]
Hi @umarahgul 👋🏼, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has some failing linters, kindly do well to fix them.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by julie-ify — COMMENTED** [review]
Hi @umarahgul,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

* You have submitted a project that has failed linters.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/modules/consumeApi.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    body: JSON.stringify({ name: gameName }),
+  };
+
+  try {
+    const response = await fetch(url, requestOptions);
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    return error;
+  }
```
Comment (VICTOR-CHUKWU) [inline src/modules/consumeApi.js:16]: - [ ] good use of try-catch block :+1:

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @umarahgul , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] arrow function used
- [ ] good use of asycn-await

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**